proto/sifnode/dispensation/v1/types.proto

```diff
@@ -51,4 +56,15 @@ message Distributions { repeated Distribution distributions = 1; }
 message Distribution {
   DistributionType distribution_type = 1;
   string distribution_name = 2;
+}
+
+message UserClaim {
+  string user_address = 1;
+  DistributionType user_claim_type = 2;
+  string user_claim_time = 3;
+  bool locked = 4;
+}
+
+message UserClaims {
```

Comment:
Can you justify why this is needed instead of just embedding repeated UserClaim in some proto?

Reply:
THis was added to keep consitency witht he way DistributionRecords and Distributions are presently declared

Reply:
Yeah I think we've done this the wrong way, will merge in as is and fix this all over in the PR it's going into.